packages/xstate-solid/README.md

```diff
@@ -0,0 +1,62 @@
+# @xstate/solid
```

Comment:
Hm, I'm not totally sure how this stuff is structured in other packages but I think that the intention, at some point, was to keep `docs/packages/*/index.md` in sync with `packages/*/README.md`. Could you copy over the more recent one to the other one?